packages/core-data/src/selectors.ts

```diff
@@ -47,7 +47,7 @@ export interface State {
 	navigationFallbackId: EntityRecordKey;
 	userPatternCategories: Array< UserPatternCategory >;
 	defaultTemplates: Record< string, string >;
-	registeredPostMeta: Record< string, { postType: string } >;
+	registeredPostMeta: Record< string, Object >;
```

Comment:
I wasn't sure if `Object` is enough or we should specify the properties. Those properties might change in the future if the REST API schema changes, and that seems unrelated to this part of the code. That's why ended up using just `Object`.

Additionally, the other similar resolvers use the same.